resource/lvm/lowlevel/exec.go

```diff
@@ -0,0 +1,109 @@
+package lowlevel
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+)
+
+type Exec interface {
+	Run(prog string, args []string) error
+	RunExitCode(prog string, args []string) (int, error) // for mountpoint querying
+	Read(prog string, args []string) (stdout string, err error)
+	ReadWithExitCode(prog string, args []string) (stdout string, rc int, err error) // for blkid querying
+
+	// unit read/write injection
+	ReadFile(fn string) ([]byte, error)
+	WriteFile(fn string, c []byte, p os.FileMode) error
+	MkdirAll(path string, perm os.FileMode) error
+	Exists(path string) (bool, error)
+}
+
+type OsExec struct {
+}
+
+func (*OsExec) Run(prog string, args []string) error {
+	log.WithField("module", "lvm").Infof("Executing %s: %v", prog, args)
```

Comment:
Please address using log everywhere: if the user needs to see the information, it should be changed to an error or status message. We are trying to avoid using log, but if it should stay as a log, please change it to debug level (instead of info). Please fix in the other locations as well.